{"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"Error.localizedDescription"},{"type":"text","text":"."}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"metadata":{"extendedModule":"Swift","symbolKind":"property","role":"symbol","title":"localizedDescription","modules":[{"name":"SwiftKyber","relatedModules":["Swift"]}],"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"roleHeading":"Instance Property","externalID":"s:s5ErrorP10FoundationE20localizedDescriptionSSvp::SYNTHESIZED::s:10SwiftKyber9ExceptionO","platforms":[{"deprecated":false,"name":"iOS","introducedAt":"8.0","unavailable":false,"beta":false},{"deprecated":false,"name":"macOS","beta":false,"introducedAt":"10.10","unavailable":false},{"beta":false,"introducedAt":"9.0","deprecated":false,"name":"tvOS","unavailable":false},{"unavailable":false,"beta":false,"deprecated":false,"introducedAt":"2.0","name":"watchOS"}]},"hierarchy":{"paths":[["doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception","doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/Error-Implementations"]]},"variants":[{"paths":["\/documentation\/swiftkyber\/exception\/localizeddescription"],"traits":[{"interfaceLanguage":"swift"}]}],"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}],"languages":["swift"]}],"kind":"declarations"}],"identifier":{"url":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription","interfaceLanguage":"swift"},"kind":"symbol","references":{"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception/Error-Implementations":{"title":"Error Implementations","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/Error-Implementations","abstract":[],"kind":"article","role":"collectionGroup","url":"\/documentation\/swiftkyber\/exception\/error-implementations","type":"topic"},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception":{"navigatorTitle":[{"text":"Exception","kind":"identifier"}],"identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception","role":"symbol","title":"Exception","type":"topic","url":"\/documentation\/swiftkyber\/exception","kind":"symbol","abstract":[{"text":"The Kyber exceptions","type":"text"}],"fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"Exception","kind":"identifier"}]},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber":{"type":"topic","role":"collection","kind":"symbol","url":"\/documentation\/swiftkyber","title":"SwiftKyber","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","abstract":[{"type":"text","text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard"}]},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception/localizedDescription":{"abstract":[],"kind":"symbol","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"localizedDescription"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"}],"type":"topic","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription","title":"localizedDescription","url":"\/documentation\/swiftkyber\/exception\/localizeddescription"}}}